samples/led/main.c

```diff
@@ -1,19 +1,11 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
-
-    // LED pattern is:
-    //   Red:   0xC = 1100 (ON,  ON,  OFF, OFF;   ON,  ON,  OFF, OFF;   ...)
-    //   Green: 0x0 = 0000 (OFF, OFF, OFF, OFF;   OFF, OFF, OFF, OFF;   ...)
-    // This means the red LED will blink slowly and the green LED is off
-    HalWriteSMBusValue(0x20, SMC_REG_LEDSEQ, FALSE, 0xC0);
+    // Set front LEDs to shine red for 2*X ms every 4*X ms
+    XSetManualLED(LED_RED, LED_RED, LED_OFF, LED_OFF);
+    // Use XSetAutoLED() to restore kernel control of the front LEDs
```

Comment:
nit: "SetAutoLED() can be used..". The word "can" highlights that this is an option, and not an implicit "FIXME"/WIP-part for the sample.